node/src/WebRtcTransport.ts

```diff
@@ -744,8 +848,10 @@ function parseIceCandidate(binary: FbsWebRtcTransport.IceCandidate): IceCandidat
 		ip         : binary.ip()!,
 		protocol   : parseProtocol(binary.protocol()),
 		port       : binary.port(),
-		type       : 'host',
-		tcpType    : binary.tcpType() === 'passive' ? 'passive' : undefined
+		type       : iceCandidateTypeFromFbs(binary.type()),
+		tcpType    : binary.tcpType() === null ?
```

Comment:
Here we are relying on `null` value. However we want it to be `undefined` when not present in the FBS object. If in the future this feature is accepted (you wrote a PR AFAIR) this is gonna fail. So wouldn't it be better if ew check `!binary.tcpType()` instead?

Reply:
TS flatbuffers does provide `null` rather than `undefined` for optionals.

> So wouldn't it be better if ew check !binary.tcpType() instead?

Nop, because the corresponding enum value can be 0, and thus such check cannot be done as you would get a false negative.

Reply:
I know that TS flatbuffers generates null instead of undefined, but we consider that unfortunate and you wrote a PR changing that, am I wrong?

Reply:
> Right. What about checking "typeof xxxx === "number"?

I have nothing agains that. We can do this in a separate PR for all optional values.

I made a [feature request](https://github.com/google/flatbuffers/issues/7656), but I'm not planning into doing the PR myself mid term.

Reply:
Ah ok, I thought you wrote a PR. We are good then.